static/app/components/globalModal/index.tsx

```diff
@@ -151,6 +151,8 @@ function GlobalModal({onClose}: Props) {
       escapeDeactivates: false,
       fallbackFocus: portal,
     });
+    ModalStore.setPauseFocusTrap(focusTrap.current.pause);
+    ModalStore.setUnpauseFocusTrap(focusTrap.current.unpause);
```

Comment:
These 2 are only called in one spot, here. And they both depend on `focusTrap.current`, what if this was one setter like `MocalStore.setFocusTrap(focusTrap.current);`